src/Microsoft.DotNet.XHarness.CLI/Commands/WebServer.cs

```diff
@@ -43,7 +43,8 @@ internal static async Task<ServerURLs> Start(TestCommandArguments arguments, ILo
                         {
                             builder.AllowAnyOrigin()
                                 .AllowAnyMethod()
-                                .AllowAnyHeader();
+                                .AllowAnyHeader()
+                                .WithExposedHeaders("*");
```

Comment:
Would this negatively impact the requests with credentials (https)? 
I'm not familiar with what ConfigureServices is intended to do and what its used for in this WebServer context.

Reply:
Answered here https://github.com/dotnet/runtime/issues/53668#issuecomment-853950940